components/director/pkg/pagination/pagination_test.go

```diff
@@ -0,0 +1,85 @@
+package pagination
+
+import (
+	"encoding/base64"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"strconv"
+	"testing"
+)
+
+func TestComputeOffset(t *testing.T) {
+	//GIVEN
+	offset := 2000
+	testCases := []struct {
+		Name            string
+		InputCursor     string
+		ExptectedOffset int
+		ExpectedErr     error
+	}{
+		{
+			Name:            "Success",
+			InputCursor:     string(base64.StdEncoding.EncodeToString([]byte(strconv.Itoa(offset)))),
```

Comment:
why don't you use EncodeOffsetCursor?

Reply:
I added tests for encode and then decode cursor